test/test-recommend.js

```diff
@@ -0,0 +1,47 @@
+const { Recommender } = require("../lib/Recommend");
+
+const tabs = require("sdk/tabs");
+function only1Tab () {
+  let first = true;
+  for (let tab of tabs) {
+    if (first) {
+      first = false;
+      continue;
+    }
+    tab.close();
+  }
+}
+
+
+exports['test only onboard while installing'] = function (assert) {
+  assert.pass();
+};
+
+
+
+exports['test onboarding' ] = function (assert) {
+  let R;
+
+  console.log(Object.keys(assert));
+
+  // install
+  R = new Recommender();
+  console.log(R.onboarding);
+  assert.ok(!R.onboarding, "R should NOT be onboarding yet.");
+  R.start('install');
+  assert.ok(R.onboarding, "R should onboarding yet.");
+  R.destroy()
+
+  // startup
+  R = new Recommender();
+  assert.ok(!R.onboarding, "R should NOT be onboarding.");
+  R.start('startup');
+  assert.ok(!R.onboarding, "R should STILL NOT be onboarding.");
+  R.destroy();
+
+
+  only1Tab();
```

Comment:
This should probably be done as a afterEach hook, like [in other tests](https://github.com/mozilla/addon-recommendation-shield-study/blob/e22bc3a976f1499e9eb0b47a3e7dcf342b432a0b/test/test-index.js#L25).